### ISSUE TITLE
new optimization tags for flambda

### ISSUE THREAD

**Issue by nilsbecker** [gasche/manual-ocamlbuild#36]
-Oclassic, -O2 etc. should these be added to the docs? or should this issue go to ocamlbuild main?

**Comment by gasche** [comment]
Yep, we should add this to the documentation. It would be good to do an ocamlbuild release shortly, for the flags to be available to users.

Would you like to send a pull-request adding the flags to the documentation?
